HardCodedStringCheckerSharp/AppController.cs

```diff
@@ -67,6 +67,8 @@ internal static bool ShouldProcessFile( string file )
             return false;
          if ( fileName.EndsWith( ".feature.cs" ) ) //This is "automatically" generated
             return false;
+         if ( fileName.EndsWith( ".steps.cs" ) ) // Acceptance Test files, not user facing
```

Comment:
The Steps files aren't generated, so they typically don't suffix with _dot_ steps, but instead just add "Steps" to the end. This is the general convention I've seen and what [SpecFlow itself does](https://github.com/techtalk/SpecFlow/tree/master/Tests/TechTalk.SpecFlow.Specs/StepDefinitions).

Reply:
Also, is this case-sensitive? Ideally it should be a case-insensitive compare.

Reply:
Yeah, my worry was that `Steps` seems a little too generic to exclude - especially since we have multi step dialogs and such, which would almost _definitely_ have user-facing strings.